Provider/PhpcrMenuProvider.php

```diff
@@ -188,7 +189,11 @@ public function get($name, array $options = array())
      */
     public function has($name, array $options = array())
     {
-        return $this->find($name, $options, false) instanceof NodeInterface;
+        try {
+            return $this->find($name, $options, false) instanceof NodeInterface;
+        } catch (NamespaceException $e) {
```

Comment:
@dbu Yes I had colon in my name (It was actually a `callable` (!) like `AcmeBundle:Menu:mainMenu`) that I was passing to `knp_menu_get`. But it seems that CmfMenuProvider was breaking the chain with a `NamespaceException`.

I'm not a Cmf/Knp expert but maybe it's better to force a `@` before the name when you want to pass a callable.

/cc @stof